mlir/docs/Tutorials/CreatingADialect.md

```diff
@@ -106,6 +106,12 @@ the LINK_COMPONENTS descriptor.  This allows cmake infrastructure to
 generate new library targets with correct linkage, in particular, when
 BUILD_SHARED_LIBS=on or LLVM_LINK_LLVM_DYLIB=on are specified.
 
+Registration of the dialect can be performed globally by editing the
+file include/mlir/InitAllDialects.h.  In this example, we can add
```

Comment:
This is framed as if it is a possibility: I believe this isn't optional, all dialects should be there.

Reply:
@joker-eph That's not true. For example, `vcix` dialect isn't registered in [`include/mlir/InitAllDialects.h`](https://github.com/llvm/llvm-project/blob/1fec981b67ac57abd4d8defd73beb5a9433c602f/mlir/include/mlir/InitAllDialects.h#L149), but you can still use the [`--test-math-to-vcix`](https://github.com/llvm/llvm-project/blob/72fb1889424fc371ef7d630009e4e579d18b9247/mlir/test/Conversion/MathToVCIX/math-to-vcix.mlir#L1) pass with `mlir-opt`.